pages/api/v1/progressReport/editProgressReportSummary.ts

```
@@ -0,0 +1,81 @@
+		const user = await authServiceInstance.GetUser(token);
+		if (!user) {
+			return res.status(401).json({ message: messages.UNAUTHORISED });
+		}
+
+		const { postId, proposalType, summary } = req.body;
+
+		if (!postId || !proposalType || !summary) {
+			return res.status(401).json({ message: messages.INVALID_PARAMS });
+		}
+
+		if (isNaN(postId) || !isProposalTypeValid(proposalType)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)